CONTRIBUTING.md

```diff
@@ -67,7 +67,21 @@ The default policy for taking contributions is “Squash and Merge” - because
 
 ### Make sure your patch works for all supported versions of swift
 
-The CI will do this for you.  You can use the docker-compose files included if you wish to check locally.  Currently all versions of swift >= 5.7 are supported.  For example usage of docker compose see the main [README](./README.md#an-alternative-using-docker-compose)
+The CI will do this for you.  Currently all versions of Swift >= 5.8 are supported.
```

Comment:
It's probably worth adding a note that the CI in the first PR requires a project maintainer to kick it off.